heartbeat/cmd/root.go

```diff
@@ -25,12 +25,15 @@ import (
 	// include all heartbeat specific autodiscovery builders
 	_ "github.com/elastic/beats/v7/heartbeat/autodiscover/builder/hints"
 
-	// register default heartbeat monitors
-	_ "github.com/elastic/beats/v7/heartbeat/monitors/defaults"
```

Comment:
This package looks broken. In the OSS magefile.go there is a generator for it. It looks like it's supposed to be creating imports for each monitor. I'd remove the package and the generation code if it's unused.

Reply:
The weird thing is this used to work, and somehow broke at some point. Now, when I run `mage update` these lines just get deleted. That said, probably easier to just remove it and make it manual since we don't add monitor types frequently.